### PR TITLE
fixing the paywall ts config

### PR DESCRIPTION
**Pull request by julien51**
# Description

This fixes storybook!

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [X] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: paywall/tsconfig.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     // "rootDir": "./",                       /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
     // "composite": true,                     /* Enable project compilation */
     "removeComments": false /* Do not emit comments to output. */,
-    //"noEmit": true,                           /* Do not emit outputs. */
+    "noEmit": true /* Do not emit outputs. */,
```
Comment (cellog) [inline paywall/tsconfig.json]:
```suggestion
    //"noEmit": true,                           /* Do not emit outputs. */
    "outDir": "/anything-other-than-.-will-work"
```

Comment (julien51) [reply paywall/tsconfig.json]: So, if I comment it out again it fails obviously...

Comment (julien51) [reply paywall/tsconfig.json]: and on all other settings the `,` is after the comment, so for consistency I'll use the same approach ;)

Comment (cellog) [reply paywall/tsconfig.json]: did you try this fix?

Comment (cellog) [reply paywall/tsconfig.json]: it succeeds here, so would be interesting if it doesn't there

Comment (cellog) [reply paywall/tsconfig.json]: and to be clear: the original PR fails locally when I build with
Screen Shot 2019-06-25 at 12 19 14 PM

**Comment by julien51** [comment]
This approach does not work. Closing until we find something better.

@cellog  will open an issue to keep track of that issue.

**Pull request closed without merging**